packages/events/defaultSubscribers/migrate/Messages.js

```diff
@@ -219,8 +221,8 @@ class Messages {
 
       deployed: () => {
         let stopText;
-        if (reporter.blockSpinner) {
-          reporter.blockSpinner.stop();
+        if (spinners.pick("migration-reporter")) {
```

Comment:
`pick` confused me because of `lodash.pick`, maybe `get` instead?

Reply:
:heavy_plus_sign:

Reply:
I just copied the interface from spinnies, as the goal was pretty much to make a static wrapper. I'll add a `get` alias, however.

FWIW, if I wanted to spend more time on this I probably would've written a more abstracted wrapper. I don't like that every operation has to be run through the top-level module functions. I wish it was just returning a `Spinner` object, for example.

Reply:
fixed in #4934